regression/goto-instrument/unwind-unwindset4/test.desc

```diff
@@ -1,6 +1,6 @@
 CORE
 main.c
---unwindset main.0:10,f.0:10,g.0:10,g.1:10 --unwinding-assertions
+--unwindset main.0:10,f.0:10 --unwindset g.0:10,g.1:10 --unwinding-assertions
```

Comment:
Commenting here, but more a comment for the PR and/or commit descriptions... But I was having a slightly hard time groking the intent of this PR until I saw this example here in the test case 👍 - So perhaps a clearer description might be something like:

```
Allow command line options that take values consisting of comma separated list to be
repeated and accumulated into a single list. For instance, making `--foo A,B,C,D` be
semantically equivalent to `--foo A,B --foo C,D`
```

Or words to that affect... not sure if its clearer or not to other people, but

Reply:
Thank you, I shamelessly copied this into the commit message!